scripts/bfb-install

```diff
@@ -27,130 +27,541 @@
 # of the authors and should not be interpreted as representing official policies,
 # either expressed or implied, of the FreeBSD Project.
 
+#
+# Configurations
+#
```

Comment:
A general comment:
The bfb-install script runs by /bin/sh (line 1), which might behave differently on Ubuntu server.
To be compatible, run "checkbashisms <this bfb-install script>" to check and fix the reported issues.
If there're issues and it's hard to fix, we could consider use /bin/bash instead of/bin/sh.

Reply:
I didn't know we can use bash :). I've changed it to bash. That makes everything much easier.